git-playground/Agustin/userinfo

```diff
@@ -1,6 +1,6 @@
-Agustin Mediotti
-06/04/1996
-Licenciatura en informatica / Analista Programador Universitario
-La Plata
-user code: agustin-1996
-month code: agustin-04
+Agustin Mediotti
+06/04/1996
+Licenciatura en informatica / Analista Programador Universitario
+La Plata
+user code: agustin-1996
+month code: agustin-04
```

Comment:
Weird, there seems to be no change

Reply:
I agree, it seems strange to me that there has been any change.